### ISSUE TITLE
календарь украины 

### ISSUE THREAD

**Issue by kotovmaksim** [xmlcalendar/data#12]
спасибо огромное за ваш труд, не первый год использую календарь для РФ
но у меня есть к вам огромное просьба, можете добавить и календарь для Украины?

**Comment by xmlcalendar** [comment]
Да, конечно, было бы здорово если бы pull request мне с ним прислали, или просто xml-к почтой

**Comment by streamteck** [comment]
Супер идея, что для этого нужно?

**Comment by xmlcalendar** [comment]
@streamteck Нужно по аналогии с российским календарем сделать для украины. Внести в него праздники и прислать на почту info@xmlcalendar.ru либо прикрепить сюда к сообщению. Либо создать pull request (см. https://rustycrate.ru/%D1%80%D1%83%D0%BA%D0%BE%D0%B2%D0%BE%D0%B4%D1%81%D1%82%D0%B2%D0%B0/2016/03/07/contributing.html)

**Comment by xmlcalendar** [comment]
@streamteck  Добавлен календарь украины 2015-2021 гг: 
http://xmlcalendar.ru/data/ua/2021/calendar.xml
http://xmlcalendar.ru/data/ua/2021/calendar.json
http://xmlcalendar.ru/data/ua/2021/calendar.csv
названия месяцев пока на русском языке
